src/main/kotlin/no/nav/familie/ef/sak/behandling/revurdering/BehandleAutomatiskInntektsendringTask.kt

```diff
@@ -153,10 +148,33 @@ class BehandleAutomatiskInntektsendringTask(
         return perioder
     }
 
-    private fun oppdaterInntektMedNyBeregnetForventetInntekt(
+    fun oppdaterInntektMedNyBeregnetForventetInntekt(
         forrigeVedtak: Vedtak,
         inntektResponse: InntektResponse,
+        revurderesFra: YearMonth,
     ): List<Inntektsperiode> {
+        if (revurderesFra.isBefore(YearMonth.now())) {
+            val inntektsperioder =
+                generateSequence(revurderesFra) { it.plusMonths(1) }
+                    .takeWhile { it.isEqualOrBefore(YearMonth.now().minusMonths(1)) }
+                    .map {
+                        Inntektsperiode(
+                            periode = Månedsperiode(it, it),
```

Comment:
Månedsperiode(it, it), - bruke konstruktør med _en_ parameter - litt lettere å lese vs it, it? Parameternavn? Kanskje it kunne fått et navn?